appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -1782,7 +1789,11 @@ public void onClick(Widget sender) {
   }
 
   private void maybeShowSplash() {
-    if (AppInventorFeatures.showSplashScreen() && !isReadOnly) {
+    String showUIPicker = userSettings.getSettings(SettingsConstants.USER_GENERAL_SETTINGS).
+                              getPropertyValue(SettingsConstants.SHOW_UIPICKER);
+    if(showUIPicker.equalsIgnoreCase("True")) {
+      new UISettingsWizard(true).show();
+    } else if (AppInventorFeatures.showSplashScreen() && !isReadOnly) {
```

Comment:
Note that if they choose Classic then the sequence of dialogs is broken here, since otherwise this would either call `createWelcomeDialog` or `maybeShowNoProjectsDialog`.

Reply:
No matter what is selected, the page reloads at this point, and the user should see Welcome/NoProjects.

Reply:
The reload is included in saveUserDesignSettings, which I'm using to save the flag that the user has been shown this dialog.